File: 1848/1848.go
func minimumDistanceToTheTargetElem(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumDistanceToTheTargetElem(root.Left)
	r := minimumDistanceToTheTargetElem(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 1848/1848.cpp
class Solution {
public:
    int minimumDistanceToTheTargetElem(TreeNode* root) {
        if(!root) return 0;
        return 1+max(minimumDistanceToTheTargetElem(root->left),minimumDistanceToTheTargetElem(root->right));
    }
};